datamade/metro-pdf-merger
# Migrate from Travis to GitHub Actions

## Description 

See title.

### Testing instructions

- Confirm CI passes for this PR.
- Merge, then confirm CI passes and a staging deployment is created and succeeds.
